crates/uv/src/commands/reporters.rs

```diff
@@ -220,6 +223,8 @@ impl ProgressReporter {
 
         let mut state = state.lock().unwrap();
 
+        let display_name = format!("{} {name}", direction.as_str());
```

Comment:
This is a good addition for the upload progress bar, but it also changes the style for e.g. `uv pip install torch`, while we want to only change the publish style